### ISSUE TITLE
Build fails

### ISSUE THREAD

**Issue by itemir** [CLCL/node-BME280#5]
Build fails with the following error:

```
> i2c@0.2.0 install /home/itemir/Temp/test/node_modules/node-bme280/node_modules/i2c
> node-gyp rebuild

make: Entering directory '/home/itemir/Temp/test/node_modules/node-bme280/node_modules/i2c/build'
  CXX(target) Release/obj.target/i2c/src/i2c.o
../src/i2c.cc: In function ‘void ReadBlock(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/i2c.cc:162:3: error: ‘MaybeLocal’ was not declared in this scope
   MaybeLocal<Object> buffer = node::Buffer::New(isolate, len);
   ^
../src/i2c.cc:162:20: error: expected primary-expression before ‘>’ token
   MaybeLocal<Object> buffer = node::Buffer::New(isolate, len);
                    ^
../src/i2c.cc:162:22: error: ‘buffer’ was not declared in this scope
   MaybeLocal<Object> buffer = node::Buffer::New(isolate, len);
                      ^
i2c.target.mk:85: recipe for target 'Release/obj.target/i2c/src/i2c.o' failed
make: *** [Release/obj.target/i2c/src/i2c.o] Error 1
make: Leaving directory '/home/itemir/Temp/test/node_modules/node-bme280/node_modules/i2c/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1078:12)
gyp ERR! System Linux 4.4.13-v7+
gyp ERR! command "node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/itemir/Temp/test/node_modules/node-bme280/node_modules/i2c
gyp ERR! node -v v0.12.15
gyp ERR! node-gyp -v v3.3.1
gyp ERR! not ok
npm ERR! Linux 4.4.13-v7+
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "--save" "node-bme280"


npm ERR! node v0.12.15


{
npm ERR! npm  v2.15.1
npm ERR! code ELIFECYCLE

npm ERR! i2c@0.2.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the i2c@0.2.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the i2c package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs i2c
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!
npm ERR!     npm owner ls i2c
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/itemir/Temp/test/npm-debug.log
```

**Comment by itemir** [comment]
Related to this: https://github.com/kelly/node-i2c/issues/79

**Comment by itemir** [comment]
Fixed by #6
